# HG changeset patch
# User "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
# Date 1322261969 -3600
# Node ID 561bce58531343b37fb1df81b6ddd56ef6762f08
# Parent  cac189eafb049bc7a3a07771ac9dfd5a516d80ba
config/target: enforce floating point support

Do not prompt for the type of floating-point support, if the
architecture did not explicitly stated that it did support it.

Reported-by: Morten Thunberg Svendsen <mts@doredevelopment.dk>
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>

diff --git a/config/target.in b/config/target.in
--- a/config/target.in
+++ b/config/target.in
@@ -260,6 +260,7 @@
 choice
     bool
     prompt "Floating point:"
+    depends on ARCH_SUPPORTS_WITH_FLOAT
 
 config ARCH_FLOAT_HW
     bool
@@ -328,6 +329,7 @@
 
 config ARCH_FLOAT
     string
+    default ""       if ! ARCH_SUPPORTS_WITH_FLOAT
     default "hard"   if ARCH_FLOAT_HW
     default "soft"   if ARCH_FLOAT_SW
     default "softfp" if ARCH_FLOAT_SOFTFP
